content/community/changelog/app-nuget/v4/whats-new/_index.nb.md

```diff
@@ -71,7 +71,7 @@ for å aktivere readiness og liveness probene.
       ```cs
       app.UseHealthChecks("/health");
       ```
-2. I `deployment/Chart.yaml` skal referansen til Studio helm charten oppdateres til versjon `2.1.0`
```

Comment:
this is an old version of the app template, not sure if we should update the what's new part here? App is currently running on v7

Reply:
Jeg ser poenget ditt - utfordringen er vel om du oppgraderer og skal se på dokumentasjon for å bruke readiness og liveness probes, så er det her de finner det. Eller vil ikke det være aktuelt?

Reply:
tror dette er siden for generell doc: https://docs.altinn.studio/app/development/configuration/deployment/
mulig vi egentlig må justere noe her og når vi først er inne på det. 
Det under changelog er vel kun aktuelt dersom du skal inn og manuelt oppgradere fordi du har fått beskjed av oss eller følger såpass med i timen at du vet vi har deployet en ny helm chart.